src/components/availability-label/availability-label.tsx

```diff
@@ -2,46 +2,60 @@ import React from "react";
 import CheckIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/Check.svg";
 import clsx from "clsx";
 import { Link } from "../utils/link";
+import { useGetAvailabilityV3 } from "../../core/fbs/fbs";
 
 export interface AvailabilityLabelProps {
   manifestText: string;
   availabilityText: string;
-  state: "available" | "unavailable" | "selected";
+  selected: boolean;
+  state: string;
   link: string | undefined;
+  materialId: string[];
 }
 
 export const AvailabilityLabel: React.FC<AvailabilityLabelProps> = ({
   manifestText,
   availabilityText,
```

Comment:
Please remove the `availabilityText` prop.

We now determine availability status from a service and have `useText()` for static texts. We can use these to set the availability text to `t('Available')` or `t('Unavailable')` depending on the response from FBS.